parcel_test.go

```diff
@@ -158,10 +159,7 @@ func TestGetByClient(t *testing.T) {
 	for _, storedParcel := range storedParcels {
 		parcel, ok := parcelMap[storedParcel.Number]
 		require.True(t, ok, "parcel with ID %d not found in parcelMap", storedParcel.Number)
-		require.Equal(t, storedParcel.Client, parcel.Client)
-		require.Equal(t, storedParcel.Status, parcel.Status)
-		require.Equal(t, storedParcel.Address, parcel.Address)
-		require.Equal(t, storedParcel.CreatedAt, parcel.CreatedAt)
+		require.True(t, reflect.DeepEqual(storedParcel, parcel), storedParcel.Number)
```

Comment:
Да, так конечно тоже можно, но это более сложная функции, тут можно и 
```suggestion
require.Equal(t, storedParcel, parcel)
```

Плюс тут не нужен третий параметр, это сообщение к ошибке, а у тебя просто номер посылки